instrumentation/spring/spring-boot-autoconfigure/build.gradle.kts

```diff
@@ -208,6 +244,15 @@ configurations.configureEach {
 tasks {
   compileTestJava {
     options.compilerArgs.add("-parameters")
+
+    // Exclude Spring Boot specific tests from compilation when testLatestDeps is true
+    // These tests are covered by testSpring4 suite
+    if (latestDepTest) {
```

Comment:
Sometimes when we have a separate suite for new version we limit the latest dep in base version to `1.+` and in version `2` suite either set the version to `2.0` or `latest.release` depending on the latest dep flag. Would that also work here? Or maybe it would help if these tests were moved to a separate suite and the main test directory would only contain tests that work on all versions?